index.html

```diff
@@ -249,5 +257,9 @@ <h2 class="head">Contact me</h2>
             <button type="submit" class="submitt" >Get in touch</button>
         </form>
     </section>
+    <!-- <script>
+        console.log(document.body.children);
+    </script> -->
+    <script src="menu.js"></script>
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.